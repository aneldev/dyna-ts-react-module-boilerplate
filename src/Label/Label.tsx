import * as React from "react";

export interface ILabelProps {
  className?: string;
  children: any;
}

export const Label = (props: ILabelProps): JSX.Element => {
  const {
    className,
    children,
  } = props;

  return (
    <div className={className}>
      {children}
    </div>
  );
};
