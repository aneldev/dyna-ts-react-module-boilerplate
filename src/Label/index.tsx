export {
  Label,
} from "./Label";
export type {
  ILabelProps,
} from "./Label";
