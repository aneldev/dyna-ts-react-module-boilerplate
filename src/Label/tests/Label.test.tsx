import {render} from '@testing-library/react';
import {Label} from "../Label";

describe('Label', () => {
  it('renders with minimal props', () => {
    const {asFragment} = render(<Label>Customers</Label>);
    expect(asFragment()).toMatchSnapshot();
  });
});
