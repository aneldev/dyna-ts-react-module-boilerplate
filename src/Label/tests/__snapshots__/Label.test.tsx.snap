// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label renders with minimal props 1`] = `
<DocumentFragment>
  <div>
    Customers
  </div>
</DocumentFragment>
`;
